myhdl/conversion/_toVHDL.py

```diff
@@ -113,6 +113,7 @@ class _ToVHDLConvertor(object):
                  "use_clauses",
                  "architecture",
                  "std_logic_ports",
+                 "no_initial_value"
```

Comment:
I'm easy with that, though I'm not sure there wasn't some thinking about default consistency in the long term. I think the plan is to ultimately make it default `False` and then `no_initial_values` makes sense.